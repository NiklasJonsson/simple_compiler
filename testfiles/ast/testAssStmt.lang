int assFun(){
	int a = b;
	int b;
	return c;
}
