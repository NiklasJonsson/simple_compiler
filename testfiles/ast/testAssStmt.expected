Program
  List
    Function
      FunctionId
      List
      Block
        List
          DeclStmt
            Opt
              IdUse
          DeclStmt
            Opt
          ReturnStmt
            IdUse
